docs/source/release/v6.6.0/diffraction.rst

```diff
@@ -10,60 +10,64 @@ Powder Diffraction
+- Fixed an issue with :ref:`HB2AReduce <algm-HB2AReduce>` when ``colltrans`` column is not available in the data file. This corresponds to the situation when the ``colltrans`` motor is not connected.
 
 
 Engineering Diffraction
 -----------------------
 
+.. image:: ../../images/6_6_release/diffraction/DiffractionUpdateEngineeringFitting.png
+    :align: right
+    :height: 500px
+
 New features
 ############
-- On the GSASII tab, when a successful refinement is run, the output table workspace for the lattice parameters now includes the Microstrain value and the column title is marked when this parameter has been refined. A similar instrument parameter table workspace is now also generated which includes Sigma-1 and Gamma(Y), with a refined flag, and the fitting range. The Pawley reflections are output to a separate table workspace if they are available.
-- An indeterminate progress bar has been added below the FitPropertyBrowser on the Engineering Diffraction Fitting tab. This will display when the fit is in progress and whether the recent fit status was success or fail.
+- On the ``GSASII`` tab of the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>`, when a successful refinement is run, the output table workspace for the lattice parameters now includes the ``Microstrain`` value and the column title is marked when this parameter has been refined. A similar instrument parameter table workspace is now also generated which includes ``Sigma-1`` and ``Gamma(Y)``, with a refined flag, and the fitting range. The Pawley reflections are output to a separate table workspace if they are available.
+- An indicator has been added below the ``FitPropertyBrowser`` on the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` ``Fitting`` tab. This will display when the fit is in progress and whether the most recent fit was successful or a failure.
 
 Bugfixes
-############
-- Correct the tabbing order between widgets on the Engineering Diffraction interface.
-- Add a scrollbar to the GSASII tab to allow a smaller interface height, like the fitting tab.
-- Check that there is enough data points before attempting to fit a diffraction peak in :ref:`FitPeaks <algm-FitPeaks>`
-- SaveVulcanGSS algorithm marked as deprecated as of 2022-11-30
+########
+- Corrected the tabbing order between widgets on the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>`.
+- Add a scrollbar to the ``GSASII`` tab, on the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>`, to allow a smaller interface height, similar to the ``Fitting`` tab.
+- A check has been added to ensure there are enough data points before attempting to fit a diffraction peak in :ref:`FitPeaks <algm-FitPeaks>`.
+- The :ref:`SaveVulcanGSS <algm-SaveVulcanGSS>` algorithm has marked as deprecated as of 2022-11-30.
```

Comment:
I think "algorithm has marked" might have been intended to be "algorithm has been marked"